applications/nrf5340_audio/src/utils/board_version.c

```diff
@@ -18,56 +18,41 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(board_version, CONFIG_BOARD_VERSION_LOG_LEVEL);
 
-#define ADC_1ST_CHANNEL_ID 0
-#define ADC_RESOLUTION_BITS 12
+#define BOARD_ID DT_NODELABEL(board_id)
+
+static const struct adc_dt_spec adc = ADC_DT_SPEC_GET(BOARD_ID);
+static const struct gpio_dt_spec power_gpios = GPIO_DT_SPEC_GET(BOARD_ID, power_gpios);
```

Comment:
What does "power_gpios" mean in this context?

Reply:
https://docs.zephyrproject.org/latest/build/dts/api/bindings/adc/voltage-divider.html

https://github.com/zephyrproject-rtos/zephyr/pull/55675/commits/f9f135531b4e124c8567181f7ea4c0320d1f6f5b